fbfmaproom/fbflayout.py

```diff
@@ -358,6 +358,16 @@ def command_layout():
                     "vertical-align": "top",
                 },
             ),
+            dbc.Alert(
+                "No forecast available for this month",
+                color="danger",
+                dismissable=True,
+                is_open=False,
+                id="forecast_warning",
+                style={
+                    "margin-bottom": "8px",
+                },
+            ),
```

Comment:
Does this block interaction until you dismiss it? If so, that seems too heavyweight. Can't we add a label in the center of the map? Maybe you could use a [tooltip](https://leafletjs.com/reference.html#tooltip) with `permanent=true`?

Reply:
It doesn't block interaction.